spring-integration-gcp/src/main/java/org/springframework/integration/gcp/outbound/PubSubMessageHandler.java

```diff
@@ -1,26 +1,29 @@
 /*
- * Copyright 2017 original author or authors.
+ *  Copyright 2017 original author or authors.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
  */
 
-package org.springframework.cloud.gcp.pubsub.integration.outbound;
+package org.springframework.integration.gcp.outbound;
+
+import java.io.IOException;
 
 import com.google.cloud.pubsub.spi.v1.Publisher;
 import com.google.pubsub.v1.PubsubMessage;
 import com.google.pubsub.v1.TopicName;
-import java.io.IOException;
-import org.springframework.cloud.gcp.pubsub.integration.converters.SimpleMessageConverter;
+
+import org.springframework.cloud.gcp.pubsub.converters.SimpleMessageConverter;
```

Comment:
Because for now the template needs it, and the template should not depend on the SI. We need to open an issue after to fix all this, for now, it's just module refactoring